src/aat/resources/features/F-029/F-029.feature

```diff
@@ -0,0 +1,65 @@
+@F-029
+Feature: F-029: Fetch an event trigger in the context of a case type
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-049
+  Scenario: must return successfully a CaseEventTrigger object filtered by case type ID
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [is prepared with valid input parameters]
+    And it is submitted to call the [Fetch an event trigger in the context of a case type for Case Worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
+    And the response has all the details as expected
+
+  @S-050
+  Scenario: must return appropriate negative response when request does not provide a valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide a valid authentication credentials]
```

Comment:
does not provide valid authentication credentials]

Reply:
updated